src/Chapter3.hs

```diff
@@ -1005,7 +1131,26 @@ properties using typeclasses, but they are different data types in the end.
 Implement data types and typeclasses, describing such a battle between two
 contestants, and write a function that decides the outcome of a fight!
 -}
-
+newtype Health' = Health' { getHealth' :: Int }
+newtype Attack' = Attack' { getAttack' :: Int }
+newtype Defense' = Defense' { getDefense' :: Int }
```

Comment:
Feel free to reuse newtypes from the previous task :slightly_smiling_face: Unfortunately, you can't multiple definitions of the same type within a single module.